benchmarks/grpo/baselines/trl_grpo.py

```
@@ -0,0 +1,103 @@
+import time
+import resource
+import argparse
+from pathlib import Path
+
+import torch
+from datasets import load_dataset
+from transformers import AutoModelForCausalLM, AutoTokenizer
+from trl import GRPOConfig, GRPOTrainer
+from peft import LoraConfig
+
+def gsm8k_reward(completions: list[str], ground_truths: list[str], **kwargs) -> list[float]:
```

Ruff (0.16.1)

[warning] Missing type annotation for `**kwargs`

(ANN003)